.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: python
+python:
+  - "2.7"
+install: pip install pyyaml
+script: make -C k8s.io all
```

Comment:
by itself, this won't do much - it'll run `make test`, but that just runs `test.py` against the production `k8s.io` site,

for this to be a meaningful presubmit, we'd probably need to deploy the configs to a test cluster and then run `test.py` against that.